README.md

```diff
@@ -145,6 +145,15 @@ Examples:
 
         curl -X PUT http://pushgateway.example.org:9091/api/v1/admin/wipe
 
+  #### MS Windows Users
+MS Windows users could send http requests to the pushgateway using Powershell. 
```

Comment:
I would leave a blank line above this for consistency. Similarly, capitalize "Pushgateway" and "HTTP".

Reply:
Final nit: Maybe "can" instead of "could"?

Reply:
```suggestion

MS Windows users can send HTTP requests to the Pushgateway using Powershell. 
```